index.html

```diff
@@ -922,25 +922,25 @@ <h2>Kontak Kami</h2>
               <div class="col-md-6">
                 <div class="info-item" data-aos="fade" data-aos-delay="200">
                   <i class="bi bi-geo-alt"></i>
-                  <h3>Address</h3>
-                  <p>A108 Adam Street</p>
-                  <p>New York, NY 535022</p>
+                  <h3>Alamat</h3>
+                  <p>Jl. Pajagalan RT 03 RW 12</p>
+                  <p>Ds. Banjaran Kulon, Kec. Banjaran, Kab. Bandung</p>
                 </div>
               </div><!-- End Info Item -->
 
               <div class="col-md-6">
                 <div class="info-item" data-aos="fade" data-aos-delay="300">
                   <i class="bi bi-telephone"></i>
-                  <h3>Call Us</h3>
-                  <p>+1 5589 55488 55</p>
-                  <p>+1 6678 254445 41</p>
+                  <h3>Hubungi</h3>
+                  <p>0821 2604 0267 (Burhanudin)</p>
```

Comment:
since this is still in development, it's better to use dummy data for sensitive areas